proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/base/ProxyNodeAppRuleTestBase.java

```diff
@@ -11,7 +11,7 @@
 import org.opensaml.security.credential.Credential;
```

Comment:
Ignore the line. Maybe we should rename this file, to be `GatewayAppRuleTestBase`

Reply:
Oh yeah good idea